web_src/js/features/imagediff.js

```diff
@@ -222,21 +222,21 @@ export function initImageDiff() {
 
       sizes.image1.css({
         width: sizes.size1.width * factor,
-        height: sizes.size1.height * factor
+        height: sizes.size1.height * factor,
       });
```

Comment:
Odd, these `.css` should be gone with https://github.com/go-gitea/gitea/pull/29852.

Reply:
Apparently the linter missed them. Will follow up later.